src/components/shared/Extensions/Modal/Modal.tsx

```diff
@@ -55,22 +55,28 @@ const Modal: FC<PropsWithChildren<ModalProps>> = ({
         {title && <h4 className="heading-5 mb-2">{title}</h4>}
         {subTitle && <p className="text-gray-600 text-md">{subTitle}</p>}
         {children}
-        <div className="flex flex-wrap gap-3 mt-8">
-          <Button mode="primaryOutline" isFullSize onClick={onClose}>
-            {closeMessage}
-          </Button>
-          <Button
-            mode={buttonMode}
-            isFullSize
-            disabled={disabled}
-            onClick={() => {
-              onConfirm?.();
-              onClose();
-            }}
-          >
-            {confirmMessage}
-          </Button>
-        </div>
+        {(closeMessage || confirmMessage) && (
+          <div className="flex flex-wrap gap-3 mt-8">
+            {closeMessage && (
+              <Button mode="primaryOutline" isFullSize onClick={onClose}>
+                {closeMessage}
+              </Button>
+            )}
+            {confirmMessage && (
```

Comment:
you checked in 58 line if props exist, checking it again is not necessary

Reply:
`{(closeMessage || confirmMessage) &&` this conditional is needed for display container `<div className="flex flex-wrap gap-3 mt-8">`

Separate conditionals `closeMessage` and `confirmMessage` are needed to display each button